src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<ValidPassword, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto, ConstraintValidatorContext context) {
+        if (userRequestDto.getPassword() == null || userRequestDto.getRepeatPassword() == null) {
```

Comment:
That's enough to check only one field on null, that you use to call method `equals`, cause under the hood `equals` checks on the `null` method parameter too 
```suggestion
        if (userRequestDto.getPassword() == null) {
```